pkg/generate/generate.go

```diff
@@ -0,0 +1,42 @@
+package generate
+
+import (
+	"bytes"
+	"io/ioutil"
+	"text/template"
+
+	"github.com/gobuffalo/packr"
+	"github.com/ocibuilder/ocibuilder/pkg/apis/ocibuilder/v1alpha1"
+)
+
+func GenerateSpecification(generator v1alpha1.SpecGenerator) error {
```

Comment:
should this be mixedCaps?

`generateSpecification`

Reply:
This is not currently be used outside of the package but will be in PR #85 and therefore should be public